spec/factories/prior_authority_application.rb

```diff
@@ -122,7 +123,9 @@
     end
 
     trait :with_further_information_supplied do
-      further_informations { [build(:further_information, :with_response, :with_supporting_documents)] }
+      after(:create) do |paa|
```

Comment:
not sure what the advantages of the changes here are??

why the `after(:create)`

Reply:
needed to do this to use the further_information objects in another let